src/videoCache.js

```diff
@@ -136,12 +136,13 @@ function shimStorageCallback(done) {
  * @param {videoCacheStoreCallback} [done] An optional callback which should be executed after
  * the data has been stored in the cache.
  */
-export function store(bids, done) {
+export function store(bids, done, isBatchRequest) {
```

Comment:
Actually, looking into this more, I don't think you need to change this at all? why couldn't you do something like (continuing from my previous comment):

```
function storeInCache(batch, auctionInstance, afterBidAdded) {
    store(batch, function(err, cacheIds){
          cacheIds.forEach(cacheId, i) {
               const bidResponse = batch[i]; // instead of the combination of `batchBidResponseRequestIdMappingObject` and `batchedBidRequestIds`
               //....
          }
    })
}
```

The way you have it here, this `batchedBidRequestIds` is just passed in and back out again - this logic does not use it at all.